app/src/app/app.module.ts

```diff
@@ -1,41 +1,36 @@
-import { AppComponent } from './app.component';
+import { HttpClientModule } from '@angular/common/http';
+import { APP_INITIALIZER, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
+
+import { AppComponent } from './app.component';
 import { EnplugService } from './enplug.service';
-import { APP_INITIALIZER, NgModule } from '@angular/core';
-import { TranslateLoader, TranslateModule, TranslateService } from '@ngx-translate/core';
-import { TranslatePoHttpLoader } from '@biesbjerg/ngx-translate-po-http-loader';
-import { HttpClient, HttpClientModule } from '@angular/common/http';
 import { translationInitializer } from './translation.initializer';
 
-export function HttpLoaderFactory(http: HttpClient) {
-  console.error('[APP SEED] Please set player-side translation url here.');
-  return new TranslatePoHttpLoader(http, '/i18n/apps/APPID/player');
-}
-
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
     BrowserModule,
     HttpClientModule,
-    TranslateModule.forRoot({
-      loader: {
-        provide: TranslateLoader,
-        useFactory: (HttpLoaderFactory),
-        deps: [HttpClient]
-      }
-    })
```

Comment:
To discuss - do we want to remove it? It's a good base for all our apps.

Reply:
As @A-Gochnio suggested, we may have another branch for ourselves. I'll prepare such branch soon.